odxtools/compumethods/compudefaultvalue.py

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import List, Optional
+from xml.etree import ElementTree
+
+from ..odxlink import OdxDocFragment
+from .compuinversevalue import CompuInverseValue
+
+
+@dataclass
+class CompuDefaultValue:
```

Comment:
CompuDefaultValue could inherit from CompuConst

Reply:
yes, but it does not do so in the XSD. given that CompuConst is pretty trivial, I prefer to stick with the XSD...

Reply:
okay, I changes my mind: it is (slightly) beneficial to derive from `CompuConst` because of the additional logic to access the value converted to the real data type. fixed: https://github.com/mercedes-benz/odxtools/pull/306/commits/cbd5a8f2232036e27181ace65dc531f67d16b0dd